ecs_grafana.tf

```diff
@@ -0,0 +1,93 @@
+resource "aws_ecs_task_definition" "grafana" {
+  count                    = local.is_management_env ? 1 : 0
+  family                   = "grafana"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "512"
+  memory                   = "4096"
```

Comment:
Don't need to do this now, but we should repeat the variable values here, so we have consistency.